### PR TITLE
cleaner sentence

### LINKED ISSUE #141: copy about speed is causing confusion

### PR DESCRIPTION
**Pull request by jolesbi**
closes #141

### CODE REVIEW

File: src/components/SectionHomeBenefits.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@ ni-section.ni-section-dark.section-home-benefits
       i.fa.fa-bolt
       .text
         .key Lightning Fast
-        .value Tendermint supports thousands of transaction per second at 1000ms latencies.
+        .value Tendermint supports thousands of transaction per second.
```
Comment (odeke-em) [inline src/components/SectionHomeBenefits.vue]: s/of transaction/of transactions/g

Comment (jolesbi) [reply src/components/SectionHomeBenefits.vue]: 😢

Comment (jolesbi) [reply src/components/SectionHomeBenefits.vue]: moving too fast, too furious 🚗 🚦 🔥

Comment (jolesbi) [reply src/components/SectionHomeBenefits.vue]: done.

Comment (odeke-em) [reply src/components/SectionHomeBenefits.vue]: #VinDieselJordan #SayTheWordFamilyEvery4Minutes

**Pull request merged**